### DISCUSSION TITLE
Switch between multiple monitors with one click only

### DISCUSSION THREAD

**Discussion by VortexOfLife** [rustdesk/rustdesk#5143] [Q&A] [answered]
At the moment I have to click the multiple monitor icon and select the other monitor to switch between monitors.
Adding a simple right click or a double click on the 1/2 icon would be much more comfortable. 
I have to switch between my monitors often so it would be faster. Adding a shortcut key also would be good.

**Comment by firefox97** [comment]
This makes rust desk unusable for me.  I also switch between 2 monitors all the time.
Teamviewer has a switch monitors icon on the collapsed menu, which works pretty well.
Currently I would click to expand the collapsed menu, then click the monitor switch button, then click the monitor I want, then another to shrink the menu again.  4 clicks to switch monitors which would normally be just a glance between physical screens is awful..
Keyboard shortcut would help.  Also I always thought that moving the  mouse to the screen edge could just switch if the monitor is to that side.  I think that would be very natural.
Please please consider doing something to make this better!!!  I am enjoying RustDesk so far, but this doesn't work for me...   Thanks!

**Reply by rustdesk** [reply, marked as answer]
https://github.com/rustdesk/rustdesk/pull/3670

![image](https://github.com/user-attachments/assets/e30711a6-4ac0-4c45-9c56-feb5c0fe915b)

**Reply by firefox97** [reply]
That does help, but I would really like those buttons to appear on the collapse menu.

**Comment by firefox97** [comment]
Nice that does help to at least have both monitor icons in the toolbar.  But what I want is the buttons to show on the hidden toolbar.

**Comment by ToninoS** [comment]
hi all, i use a Logitech MX Anywhere 2S mouse on my laptop and it has two extra thumb buttons that i use to switch my laptop virtual screens scrolling either left or right through screens... i use 3 virtual screens so it is similar workflow that i have gotten use to on my desktop. And, like most i have my opinions so...  what would be really cool is to setup those two buttons so that they act the same way if the rust desk window is the active window. i think this may work as i have noticed when the rust desk window is active my thumb buttons dont actually scroll my laptop screens anyway as the mouse commands are going to the desktop so maybe that would be a great solution within rust desk? cheers
